gst/nnstreamer/tensor_query/tensor_query_client.c

```diff
@@ -340,14 +344,21 @@ gst_tensor_query_client_sink_event (GstPad * pad,
 
       if (gst_tensors_config_validate (&self->in_config)) {
         /** Request out config to server_src. */
-        TensorQueryCommandData cmd_buf;
 
         self->src_conn =
             nnstreamer_query_connect (self->protocol, self->src_host,
             self->src_port, DEFAULT_TIMEOUT_MS);
         if (!self->src_conn) {
           nns_loge ("Failed to connect server source ");
-          return FALSE;
+          return ret;
+        }
+        cmd_buf.cmd = _TENSOR_QUERY_CMD_CLIENT_ID;
+        id = g_uuid_string_random ();
```

Comment:
uuid is supported from glib version 2.52.
If the version is too recent, I will generate the id as a manual.
@jaeyun-jung @again4you